packages/x-components/src/x-modules/facets/components/filters/__tests__/editable-number-range-filter.spec.ts

```diff
@@ -15,11 +15,13 @@ import { XPlugin } from '../../../../../plugins/x-plugin';
 import { facetsXModule } from '../../../x-module';
 import { resetXFacetsStateWith } from '../../__tests__/utils';
 
-Object.defineProperty(HTMLInputElement.prototype, 'valueAsNumber', {
-  get() {
-    return parseFloat(this.value);
-  }
-});
+if (!HTMLInputElement.prototype.valueAsNumber) {
```

Comment:
WTF!?!?

Reply:
Why was it working before?

Reply:
It was working because the polyfill was only defined once, and had `configurable: false`, but as no test was trying to override it, there was no problem at all. 

The part that I still do not understand is that the JSDOM prototypes are shared between tests somehow, and it only happens in the CI. In our laptops, both @davidmfempathy and me tried running the tests with different versions of node, and they always worked, but in GitHub, it looked like they were sharing the `valueAsNumber` property polyfill, so when the second test was launched this property was already defined, and because of the `configurable: false` default value, it failed defining the polyfill.